packages/manager/apps/web/client/app/domain/anycast/terminate/confirm/confirm.routing.js

```diff
@@ -0,0 +1,14 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.domain.product.anycast-terminate.confirm', {
+    url: '/confirm',
+    views: {
+      modal: {
+        component: 'domainAnycastConfirmTerminate',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
```

Comment:
what if the user bookmarks this URL and visits directly using the link later. Will the goBack  work in this case?

Reply:
yes it works

Reply:
@radireddy `^` stands for parent state (https://ui-router.github.io/ng1/docs/latest/classes/state.stateservice.html#go)